src/bin-support.js

```diff
@@ -1,8 +1,14 @@
 'use strict';
 
-const DEFAULT_EXTENSIONS = 'js,ts';
+const DEFAULT_JS_EXTENSIONS = 'js,ts';
 
-async function runTransform(binRoot, transformName, args, extensions = DEFAULT_EXTENSIONS) {
+function getTransformPath(binRoot, transformName) {
+  const path = require('path');
+
+  return path.join(binRoot, '..', 'transforms', transformName, 'index.js');
```

Comment:
Can we do `require.resolve(path.join(binRoot, '..', 'transforms', transformName))`?

Reply:
Done